instrumentation/jetty-httpclient/jetty-httpclient-9.2/testing/src/main/groovy/io/opentelemetry/javaagent/instrumentation/jetty/httpclient/v9_2/AbstractJettyClient9Test.groovy

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_2
+
+import io.opentelemetry.api.common.AttributeKey
+import io.opentelemetry.context.Context
+import io.opentelemetry.context.Scope
+import io.opentelemetry.instrumentation.test.base.HttpClientTest
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes
+import org.eclipse.jetty.client.HttpClient
+import org.eclipse.jetty.client.api.ContentResponse
+import org.eclipse.jetty.client.api.Request
+import org.eclipse.jetty.client.api.Response
+import org.eclipse.jetty.client.api.Result
+import org.eclipse.jetty.http.HttpMethod
+import org.eclipse.jetty.util.ssl.SslContextFactory
+import org.junit.Rule
+import org.junit.rules.TestName
+import spock.lang.Shared
+
+import java.util.concurrent.TimeUnit
+
+abstract class AbstractJettyClient9Test extends HttpClientTest<Request> {
+
+  abstract HttpClient createStandardClient()
+
+  abstract HttpClient createHttpsClient(SslContextFactory sslContextFactory)
```

Comment:
The HTTPS client will work with HTTP too right? Do we need two separate ones?

Reply:
HTTPS client should work for non-https.  but creation of both to put the both styles through the testing process